### PR TITLE
tests: sweep extended tests for referenced to origin

### PR DESCRIPTION
**Pull request by damemi**
Trying to fix up https://github.com/openshift/origin/pull/23091

### CODE REVIEW

File: test/extended/util/framework.go [hunk 1]
```diff
@@ -1347,9 +1342,10 @@ func CreateExecPodOrFail(client corev1client.CoreV1Interface, ns, name string) s
 // CheckForBuildEvent will poll a build for up to 1 minute looking for an event with
 // the specified reason and message template.
 func CheckForBuildEvent(client corev1client.CoreV1Interface, build *buildv1.Build, reason, message string) {
+	scheme, _ := apitesting.SchemeForOrDie(buildv1.Install)
```
Comment (damemi) [inline test/extended/util/framework.go]: @mfojtik trying this to get away from legacyscheme

**Comment by mfojtik** [comment]
@damemi thanks! continuing in https://github.com/openshift/origin/pull/23138 :-)

**Pull request closed without merging**